ACMAS/app/ACMAS_Web/views.py

```diff
@@ -114,12 +96,12 @@ def returnQuery(request):
     )
 
 
-@csrf_protect
 def pdfReader(request):
     name = request.GET.get("url")
     # If the file is a .txt load by filename
     if name[-4:] == ".txt":
```

Comment:
I think this works for getting file extensions more generally,
```
if '.' in name:
  file_extension = name.rsplit('.')[1]
```

Reply:
Implemented